src/OpenFeature/OpenFeature.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using OpenFeature.Model;
+
+namespace OpenFeature
+{
+    public sealed class OpenFeature
+    {
+        private EvaluationContext _evaluationContext;
+        private IFeatureProvider _featureProvider = new NoOpFeatureProvider();
+        private readonly List<IHook> _hooks = new List<IHook>();
+        //TODO Add ILogger interface so we use the standard logging interface

```

Comment:
Perhaps relevant: https://github.com/open-feature/spec/issues/39

Reply:
Yea 100%, my thought has we would provide a method to set the [ILogger](https://docs.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line) on the OpenFeature singleton and then we would just log using that. This allow the end users to easily integrate with their choice of logger (MS, Serilog, NLog, etc)